extensions/git/src/historyProvider.ts

```diff
@@ -33,6 +37,8 @@ export class GitHistoryProvider implements SourceControlHistoryProvider, IDispos
 		this._onDidChangeCurrentHistoryItemGroup.fire();
 	}
 
+	private historyItemDecorations = new Map<string, FileDecoration>();
+
 	private disposables: Disposable[] = [];
```

Comment:
```suggestion
	private readonly disposables: Disposable[] = [];
```

Reply:
Thank you! Will get this in, in the next PR...